### ISSUE TITLE
Add metrics to Lora and Sigfox adapters

### ISSUE THREAD

**Issue by sophokles73** [eclipse/hono#1593]
The Lora and Sigfox adapters should also report the metrics that Hono's other adapters support.

**Comment by thiyagu-7** [comment]
If this is up for grabs, can I take it?

**Comment by sophokles73** [comment]
@thiyagu-7 yes, that is what the *help wanted* label is supposed to indicate. Thanks for your interest in contributing to Hono :+1:
